### PR TITLE
049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService…

### PR DESCRIPTION
**Pull request by istoimenova**
Script is part of TS: #1163
Script is common as PRs: #1326, #1327

### CODE REVIEW

**Comment by istoimenova** [comment]
@dboltovskyi, please review.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua [hunk 1]
```diff
@@ -0,0 +1,170 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PTU] [GENIVI] PolicyTableUpdate has NO "certificate" and "ForceProtectedService"=OFF at .ini file
+-- [PTU] [GENIVI] SDL must start PTU for any app except navi right after app successfully request to start first secure service
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @istoimenova, Seems "[PTU] [GENIVI] PolicyTableUpdate is valid and brings "certificate"" req-t is missed, isn't it?

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    end)
+
+  EXPECT_HMICALL("UI.SetAudioStreamingIndicator"):Times(0)
+  self.mobileSession:ExpectEvent(startserviceEvent, "Service 7: RPC SetAudioStreamingIndicator"):Times(0)
+
+  commonTestCases:DelayedExp(10000)
+end
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @istoimenova, Please rename section to "Test"

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua (hunk 1 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @dboltovskyi, thank you for notice. Done in 5ba86daa08d2a7bdfd485dacc011a2d550964b1a

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua (hunk 2 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @dboltovskyi, done in 5ba86daa08d2a7bdfd485dacc011a2d550964b1a

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+function Test:Precondition_First_StartService()
+  self.mobileSession.correlationId = self.mobileSession.correlationId + 1
+  local msg = {
+    serviceType = 7,
+    frameType = 0,
+    frameInfo = 1,
+    encryption = true,
+    rpcCorrelationId = self.mobileSession.correlationId
+  }
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @istoimenova, Are we expecting to start just RPC service without SetAudioStreamingIndicator? If so, description of the test script has to be updated

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+    return ( (data.serviceType == 7) and (data.frameInfo == 2 or data.frameInfo == 3) )
+  end
+
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", { status = "UPDATE_NEEDED", {status = "UPDATING"}}):Times(2)
+  EXPECT_HMICALL("BasicCommunication.PolicyUpdate")
+  :Do(function(_,data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+    
+  self.mobileSession:ExpectEvent(startserviceEvent, "Service 7: RPC SetAudioStreamingIndicator"):Times(0)
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @istoimenova, If we are starting just RPC service without sending of SetAudioStreamingIndicator this line has to be updated

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+-- -> SDL should trigger PTU: SDL.OnStatusUpdate(UPDATE_NEEDED)
+-- -> SDL should not respond to StartService_request
+-- -> SDL should not process request to HMI
+--
+-- 2. Performed steps
+-- Send correct policy file, certificate exists in module_config
+--
+-- Expected result:
+-- 1. SDL sends SDL.OnStatusUpdate(UP_TO_DATE)
+-- 2. SDL process request to HMI
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @istoimenova, Please update if sending of SetAudioStreamingIndicator is not required

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+--
+-- 1. Used preconditions:
+-- RPC SetAudioStreamingIndicator is allowed by policy
+-- ForceProtectedService is set to OFF in .ini file
+-- Default app exists in LP, no certificate in module_config
+-- Register and activate application.
+-- Send StartService(serviceType = 7 (RPC))
+-- -> SDL should trigger PTU: SDL.OnStatusUpdate(UPDATE_NEEDED)
+-- -> SDL should not respond to StartService_request
+-- -> SDL should not process request to HMI
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @istoimenova, Please update if sending of SetAudioStreamingIndicator is not required

Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @dboltovskyi, done

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua (hunk 5 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @dboltovskyi, done

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua (hunk 3 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @dboltovskyi, yes we expect only StartService for RPC, without SetAudioStreamingIndicator. Specification for SetAudioStreamingIndicator in description is removed.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua (hunk 4 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @dboltovskyi, done

**Pull request merged**